modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/layout-page-templates/display-page-templates/app-detail/page-definition.json

```
@@ -116,7 +115,7 @@
 												"definition": {
 													"fragment": {
 														"key": "image",
-														"siteKey": "[$GROUP_KEY$]"
+														"siteKey": "Marketplace"
```

Please make the replacement
```suggestion
														"siteKey": "[$GROUP_KEY$]"
```